index.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE>
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Salmon Cookies</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <section id="main-content">
+      <h1>Salmon Cookies</h1>
+        <p>They're cookies made into the shape of a salmon, and actually made with salmon (though the fish is ground up so fine that you can't even taste it!) that is the ideal match for a steaming cup of coffee on a gray Seattle morning. Or gray Seattle afternoon... </p>
```

Comment:
Why is this line indented more than the h1? Also, can you think of a cleaner way to format this paragraph?